src/org-manager/containers/pending-pbas/pending-pbas.component.spec.ts

```diff
@@ -92,32 +92,36 @@ describe('PendingPBAsComponent', () => {
       fixture.detectChanges();
     });
 
-    it('should show a table with a single row that says there are none pending', () => {
-      const tds = utils.getDebugElements('td');
-      expect(tds.length).toEqual(1);
-      utils.checkText(tds[0], 'There are no pending PBAs');
+    it('should show there are no new PBA requests.', () => {
+      component.loaded$ = of(true);
+      fixture.detectChanges();
+      const messageContent = fixture.debugElement.nativeElement.querySelector('.govuk-heading-l').parentElement.textContent;
+      expect(messageContent).toContain('There are no new PBA requests.');
     });
 
     it('should show an appropriate count of zero in the title', () => {
       utils.checkText(utils.getTitle(), 'New PBAs (0)');
+
       expect(component.pendingPBAsCount).toEqual(0);
     });
 
     it('should not show a "Loading..." indicator', () => {
+      component.loaded$ = of(true);
+      fixture.detectChanges();
       expect(utils.getLoadingIndicator()).toBeNull();
     });
   });
 
   describe('When PBAs have been loaded', () => {
     let storeDispatchSpy: jasmine.Spy;
     const ORGS_WITH_PENDING_PBAS: OrganisationModel[] = [
-      utils.organisationModel('bob', [ 'PBA0000001', 'PBA0000002' ]),
```

Comment:
Formatting issue again - the spaces need to be in there otherwise it will look like you are amending lines you haven't touched.

Reply:
As discussed - it appears that different people are using different types of formatting. In this case I see you've addressed the issue, so resolved.